terraform/workspace-variables/research.tfvars

```diff
@@ -0,0 +1,38 @@
+# Platform
+region                      = "eu-west-2"
+parameter_store_environment = "dev"
+app_environment             = "research"
+environment                 = "research" # For review, pass in TF_VAR_environment=review-pr-[PR number]
+
+# CloudFront
+distribution_list = {
+  "tvsresearch" = {
+    offline_bucket_domain_name    = "530003481352-offline-site.s3.amazonaws.com"
+    offline_bucket_origin_path    = "/teaching-vacancies-offline"
+    cloudfront_origin_domain_name = "teaching-vacancies-research.london.cloudapps.digital"
+  }
+}
+
+channel_list = {
+  #   "tvsreview" = {
+  #     cloudwatch_slack_channel = "twd_tv_dev"
+  #   }
+}
+
+# Documents S3 bucket
+documents_s3_bucket_force_destroy = true
+
+# Gov.UK PaaS
```

Comment:
Some of the variables have the same values as defaults, they can be removed for clarity